### PR TITLE
Remove file headers

### PR DESCRIPTION
**Pull request by ghostwriter**
**[QA]** Remove file header (DocBlock comments)

### CODE REVIEW

**Comment by boesing** [comment]
Hm, this would be mergable but codestandard has some problems with a few files.

Just ping me here when fixed. 👍🏼

**Comment by ghostwriter** [comment]
@boesing ready for review.

**Review by boesing — APPROVED** [review]

File: src/autoload.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:disable PSR12.Files.FileHeader.IncorrectOrder,PSR12.Files.FileHeader.SpacingAfterBlock,WebimpressCodingStandard.Files.DeclareStrictTypes.BelowComment
```
Comment (ghostwriter) [inline src/autoload.php:1]: @weierophinney hey, does this fix this issue?

https://github.com/laminas/laminas-coding-standard/blob/1897419d40e2e8b3f66e4f1d2e7cd690b5efd832/src/LaminasCodingStandard/ruleset.xml#L105-L110

Comment (weierophinney) [reply src/autoload.php:1]: Likely, but it wasn't present in the version in the lockfile, so the disable rule above takes care of it.

**Pull request merged**